docs/programming_guide/experiment_tracking/experiment_tracking_log_writer.rst

```diff
@@ -0,0 +1,152 @@
+.. _experiment_tracking_log_writer:
+
+##############################
+Experiment Tracking Log Writer
+##############################
+
+.. note::
+
+    This page covers experiment tracking using :class:`LogWriters <nvflare.app_common.tracking.log_writer.LogWriter>`,
+    which are configured and used with :ref:`executor` or :ref:`model_learner` on the FLARE-side code.
+    If using the Client API, please refer to :ref:`experiment_tracking_apis` for adding experiment tracking to your custom training code.
+
+***********************
+Overview and Approaches
+***********************
+
+In a federated computing setting, the data is distributed across multiple devices or systems, and training is run
+on each device independently while preserving each client's data privacy.
+
+Assuming a federated system consisting of one server and many clients and the server coordinating the ML training of clients,
+we can interact with ML experiment tracking tools in two different ways: 
+
+    - Client-side experiment tracking: Each client will directly send the log metrics/parameters to the ML experiment
+      tracking server (like MLflow or Weights and Biases) or local file system (like tensorboard)
+    - Aggregated experiment tracking: Clients will send the log metrics/parameters to FL server, and the FL server will
+      send the metrics to ML experiment tracking server or local file system
+
+Each approach will have its use cases and unique challenges. In NVFLARE, we developed a server-side approach (in the
+provided examples, the Receiver is on the FL server, but it could also be on the FL client):
```

Comment:
Why is this called the server-side approach?
The Receiver can be configured on either FL server or FL client, could also be both.